src/dbus.c

```diff
@@ -1,15 +1,14 @@
 /* copyright 2013 Sascha Kruse and contributors (see LICENSE for licensing information) */
-
-#include <stdlib.h>
-#include <stdio.h>
+#include "dbus.h"
```

Comment:
I don't feel like "dbus.h" is an import here, because "dbus.c" is the one implementing it. It's more like an export.

Anyway, if it's moved down, how should it be ordered relative to other local includes? Remain the first one or just sorted lexicographically?